src/System.Linq.Queryable/src/System.Linq.Queryable.csproj

```diff
@@ -32,5 +32,12 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Linq\src\System.Linq.csproj">
```

Comment:
I couldn't get a build using `msbuild /t:RebuildAndTest` to reference it without it. Perhaps I am not building in the most appropriate way? Or not referencing in the most appropriate way?